package.json

```diff
@@ -20,7 +20,10 @@
     "prop-types": "^15.7.2",
     "react": "17.0.1",
     "react-dom": "17.0.1",
-    "sass": "^1.30.0"
+    "react-hook-form": "^6.15.5",
+    "react-modal": "^3.12.1",
+    "sass": "^1.30.0",
+    "validator": "^13.5.2"
```

Comment:
if we are using a package `react-hook-form` then I dont think this is needed. What do you think?

Reply:
I guess sass is a part of pre-existing code, and it's being shown as a new commit because I recently merged (and re-synced) the rds to my forked one. I don't remember installing and importing sass